### PR TITLE
[Components with more than 1 service] LPS-188896 Refactor XLIFFInfoFormTranslationImporter component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related issue: https://liferay.atlassian.net/browse/LPS-188896

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:lima

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:lima

**Comment by Valmir-unicap** [comment]
ci:test:startup

File: modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/util/XLIFFTranslationImporterUtil.java [hunk 1]
```diff
@@ -0,0 +1,638 @@
...
+import com.liferay.portal.kernel.service.LayoutLocalServiceUtil;
+import com.liferay.portal.kernel.util.FileUtil;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.Validator;
+import com.liferay.portal.kernel.xml.Document;
+import com.liferay.portal.kernel.xml.DocumentException;
+import com.liferay.portal.kernel.xml.SAXReaderUtil;
+import com.liferay.segments.model.SegmentsExperience;
+import com.liferay.segments.service.SegmentsExperienceLocalServiceUtil;
```
Comment (dantewang) [inline modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/util/XLIFFTranslationImporterUtil.java:36]: @rafaprax Do we have any preference on Utils over helper components? I have a concern about replacing all these hard references with utils/snapshots but I'm not very sure.

Comment (rafaprax) [reply modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/util/XLIFFTranslationImporterUtil.java:36]: We do not have any preference as far as I know. However, I talked to @Valmir-unicap some days ago about this fix and we decided to follow another approach that will not include a Util class for this case. 

PS: I didn't write this request here for some reason that I do not remember. Sorry and thanks

**Comment by rafaprax** [comment]
@Valmir-unicap please resend when you finish the fix thanks

**Pull request closed without merging**